src/main/java/thePackmaster/cards/serpentinepack/ColdCalculated.java

```diff
@@ -0,0 +1,48 @@
+package thePackmaster.cards.serpentinepack;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.actions.utility.WaitAction;
+import com.megacrit.cardcrawl.actions.watcher.ChangeStanceAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.VulnerablePower;
+import thePackmaster.powers.bitingcoldpack.FrostbitePower;
+import thePackmaster.stances.serpentinepack.CunningStance;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class ColdCalculated extends AbstractSerpentineCard {
+
+    private static final int COST = 1;
+    private static final int MAGIC = 4;
+    private static final int DAMAGE = 7;
+    private static final int UPG_DAMAGE = 2;
+    private static final int UPG_MAGIC = 2;
+    public final static String ID = makeID("ColdCalculated");
+
+
+    public ColdCalculated() {
```

Comment:
This seems weak for a rare card. It's values are just a tad higher than a uncommon attack from the frostbite pack and that doesn't exhaust. Entering Cunning Stance doesn't seem too highly values since it's also on a common non-exhausting card